charts/seed-controlplane/charts/etcd/templates/etcd-statefulset.yaml

```diff
@@ -14,30 +14,6 @@ spec:
     app: etcd-statefulset
     role: {{ .Values.role }}
   ports:
-  - name: discovery
-    protocol: TCP
-    port: 2380
-    targetPort: 2380
----
-apiVersion: v1
```

Comment:
Yes. We use headless service at port 2379. Service @2380 was for peer communication which we do not need. Prometheus knows how to scrap from etcd instances because we are providing it with the headless endpoints in `scrap_config`.